tests/ffi/main.fmf

```diff
@@ -0,0 +1,8 @@
+/:
```

Comment:
`/: ` is this required?

Reply:
In general looks good

Reply:
Yes, /: with inherit: false, it is the fmf syntax.